packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx

```diff
@@ -20,6 +20,9 @@ import { checkCode, getCode } from '../../lib/totp';
 import { HomePath } from '../../constants';
 import { cloneDeep } from '@apollo/client/utilities';
 import { alertTextExternal } from '../../lib/cache';
+import { logViewEvent, useMetrics } from '../../lib/metrics';
+
+export const metricsPreInPostFix = 'settings.two-step-authentication';
```

Comment:
this var name is cryptic to me but 🤷

Reply:
That string can be a pre-, in-, or post-fix, so yeah 🤷